CONTRIBUTING.md

```diff
@@ -89,6 +89,8 @@ GPU Docker: `docker run --runtime=nvidia --rm -it -v ${PWD}:/addons -w /addons g
 
 Configure:
 ```
+# Temporary until we remove py2 support
+ln -sf /usr/bin/python3.6 /usr/bin/python && rm /usr/bin/python2 
```

Comment:
Would like to be able to remove this. The problem is that `configure.sh` will install using `python`:
https://github.com/tensorflow/addons/blob/master/configure.sh#L98

If we make that `python3` we should be able to remove this instruction. Bazel >1.0 will by default run tests/builds on python3 I believe

Reply:
I removed it. The issue is that in `configure.sh`, I cannot call `python3` because on windows it doesn't work like that. There is only `python` available, and `python3` doesn't exist. I believe we need to keep the symlink until `python` calls python3 by default on ubuntu.

Reply:
We don't have any way to removing it without breaking the build. Windows doesn't understand the `python3` command and on ubuntu `/usr/bin/python` is still py2 for the time being.